sdk/search/Azure.Search.Documents/README.md

```diff
@@ -249,8 +249,8 @@ SearchResults<SearchDocument> response = client.Search<SearchDocument>("luxury")
 foreach (SearchResult<SearchDocument> result in response.GetResults())
 {
     SearchDocument doc = result.Document;
-    string id = (string)doc["hotelId"];
-    string name = (string)doc["hotelName"];
+    string id = (string)doc["HotelId"];
```

Comment:
But all the other samples above use camelCase field names? I thought the plan was to document switching over to the public read-only Hotels sample, which should apply for the whole of the README samples, code and markdown.

Reply:
That is still the plan.  It's just not happening in time for this release so I'm fixing up the README manually in the short term.  The other samples with field names are all creating an index which might not perfectly match the same index you'd get from the Portal, but isn't going to lead to any bugs.